sdk/js-cloud-server/src/index.ts

```diff
@@ -0,0 +1,41 @@
+import { DevCycleOptions } from './types'
+import { DevCycleCloudClient } from './cloudClient'
+import { isValidServerSDKKey } from './utils/paramUtils'
+import { DevCycleUser } from './models/user'
+
+export { DevCycleCloudClient, DevCycleUser }
+export * from './models/populatedUser'
+export * from './models/user'
+export * from './models/variable'
+export * from './types'
+export * from './request'
+export * from './utils/logger'
+export * from './utils/paramUtils'
+
+type DevCycleOptionsCloudEnabled = DevCycleOptions & {
+    enableCloudBucketing: true
+    platform?: 'NodeJS' | 'Electron' | 'EdgeWorker'
+    platformVersion?: string
+    hostname?: string
+}
+
+export function initializeDevCycle(
+    sdkKey: string,
+    options: DevCycleOptionsCloudEnabled = { enableCloudBucketing: true },
```

Comment:
why is it still an option to "enableCloudBucketing" or not?

Reply:
shouldn't we just not have this option available here?

Reply:
yea I was originally thinking I would import this function in NodeJS, but didn't end up doing that.